### DISCUSSION TITLE
Bticino Living Now e Home Assistant su una nuova casa a 3 piani

### DISCUSSION THREAD

**Discussion by jovannig** [indomus/forum#820] [HUB: Home Assistant]
Ho cercato questa info su molti forum, ma non sono riuscito a levarci le gambe.
Ho acquistato una nuova casa disposta su 3 piani (taverna, piano terra e primo piano) in cui gli impianti elettrici sono da fare.
Ho un Raspberry 4 su cui ho installato HA core su docker, da dove vorrei controllare la smart home. Ho anche acquistato la Conbee II.
Per l'impianto elettrico mi sono orientato sulla linea Living Now (non ho ancora chiare le differenze con la linea Living Now Smart).
Prenderei anche il gateway K4500c in modo da poter effettuare gli aggiornamenti firmware sulle varie prese / deviatori.
A piano terra (quello centrale) ho il router wifi, il raspberry e anche il gateway K4500c. Essendo la casa su piu' piani, sono sicuro che i vari dispositivi bticino facciano da bridge e portino il segnale zigbee alle altre stanze sugli altri piani?
Ho anche alcune luci Philips Hue (con un gateway philips, sempre a piano terra). 
Ho anche acquistato vari sensori di movimento Philips e Aqara (senza gateway Aqara) e altri sensori di umidita' e temperatura Xiaomi (senza gateway ma su protocollo zigbee) che andranno posizionate su tutti i piani.
Essendo un po' acerbo sull'argomento chiedo a voi consigli su come muovermi, prima che l'impianto elettrico sia messo in opera.

Vi ringrazio anticipatamente
Giovanni

**Comment by ganaweb** [comment]
Ciao,
io pure ho l'impianto bTicino Living Now connesso ad Home Assistant tramite chiavetta ConBee II.
Ti confermo che le varie prese e deviatori connessi fanno da bridge portando il segnale ZigBee anche in zone remote della casa.
Tutti i sensori che hai acquistato, se di tecnologia zigbee, possono essere tranquillamente associati con la chiavetta ConBee II.

Io personalmente ti consiglierei di configurare Zigbee2MQTT.
Ci sono le guide qui su Indomus.
Evita di usare Zigbee Home Automation (ZHA) o altro.

Ciao.

**Reply by jovannig** [reply]
Grazie per la risposta. L'unico dubbio che ho e' sui sensori Aqara e Xiaomi. Avro' bisogno di installare un loro gateway oppure si allacciano ai deviatori / bridge della bticino? 
Hai per caso installato anche il termostato bticino? Oppure altri accessori della bticino che secondo te sono utili? Grazie ancora!

**Reply by ganaweb** [reply]
> Grazie per la risposta. L'unico dubbio che ho e' sui sensori Aqara e Xiaomi. Avro' bisogno di installare un loro gateway oppure si allacciano ai deviatori / bridge della bticino? 
> Hai per caso installato anche il termostato bticino? Oppure altri accessori della bticino che secondo te sono utili? Grazie ancora!

I gateway Aqara e Xiaomi non servono. Se sono zigbee fanno una rete unica con i prodotti bticino. Io ho installato anche lampadine, button e sensori della domotica di Ikea. Funzionano perfettamente senza avere il gateway Ikea.
Ho installato anche il termostato. Mi ci trovo molto bene. Su Home assistent ho il grafico con la temperatura attuale e la temperatura richiesta sul termostato. Puoi regolarlo tranquillamente da remoto.

**Reply by jovannig** [reply]
Io al momento uso deconz, sono alle prime armi con home assistant. Tu usi zigbee2mqtt pel la compatibilita' con bticino?

**Reply by snakuzzo** [reply]
Guarda [QUI](https://www.zigbee2mqtt.io/information/supported_devices.html#v=BTicino)
E se un device non è presente c'è sempre la procedura per farli aggiungere [QUI](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html)

**Reply by jovannig** [reply]
Ma per aggiungere le varie prese connesse /interruttori connessi, ecc. c'e' bisogno di qualche integrazione oppure funziona tutto con zigbee2mqtt?

**Reply by snakuzzo** [reply]
Che intendi ?
zigbee2mqtt per funzionare necessità di un broker MQTT, che sarà lo stesso broker a cui si collegherà Home Assistant (tramite [integrazione MQTT](https://www.home-assistant.io/integrations/mqtt/)).
Fatto questo, vale quello che già ti ho scritto nel [precedente post](https://github.com/indomus/forum/discussions/820#discussioncomment-1486722)

**Reply by jovannig** [reply]
Ho gia' sia Zigbee2mqtt e Mosquitto su 2 docker container, su un altro ho home assistant. Non avendo mai configurato niente della bticino vorrei sapere se tutti gli interruttori vengono visti subito da zigbee2mqtt..

**Reply by snakuzzo** [reply]
Ripeto per l'ennesima volta... la risposta alla tua domanda è [QUI](https://github.com/indomus/forum/discussions/820#discussioncomment-1486722)

**Reply by jovannig** [reply]
Non sono un esperto, ma a quanto ho capito sia gli interruttori connessi che le prese connesse sono tutte compatibili.
Ho anche un'altra domanda: nella taverna ho come illuminazione soltanto 3 plafoniere hue surimu. 
In teoria gli interruttori potrebbero non servire. L'elettricista mi ha proposto di mettere 3 interruttori normali on/off non connessi che abilitano/disabilitano le plafoniere. Pero' puo' ad esempio capitare che qualcuno voglia accendere la plafoniera utilizzando un interruttore. Cosa mi consigliate di fare?

**Reply by jovannig** [reply]
Inoltre, alla configurazione iniziale sull'app Home+Control della Legrand, avendo iPhone conviene procedere con la configurazione tramite homekit oppure seguire un'alta via, visto che poi usero' Home Assistant?
